templates/settings-mongodb.py.erb

```diff
@@ -0,0 +1 @@
+PULP2_MONGODB = {"name": "<%= scope['pulpcore::mongo_db_name'] %>", "seeds": "<%= scope['pulpcore::mongo_db_seeds'] %>", "ssl": <%= scope['pulpcore::mongo_db_ssl'] %>}
```

Comment:
I'd suggest so format this in multiple lines.

Reply:
Will do. I'm planning as well to put this back in the main `settings.py.erb` template with a conditional.